libwacom/libwacom.c

```diff
@@ -1206,6 +1206,18 @@ libwacom_stylus_get_name (const WacomStylus *stylus)
 	return stylus->name;
 }
 
+LIBWACOM_EXPORT int
+libwacom_stylus_get_num_paired_ids(const WacomStylus *stylus)
+{
+	return stylus->num_ids;
+}
+
+LIBWACOM_EXPORT const int *
+libwacom_stylus_get_paired_ids(const WacomStylus *stylus)
+{
+	return stylus->paired_ids;
+}
+
```

Comment:
IMO better:
```c
const int *libwacom_stylus_get_paired_ids(const WacomDevice *device, int *num_paired_ids);
```
That's the prototype we use for `libwacom_get_supported_styli` already. And I can't think of a use-case where we only need the num_paired anyway, so having just one function here is better.

Reply:
ACK